pygeos/creation.py

```diff
@@ -125,45 +125,51 @@ def linearrings(coords, y=None, z=None, indices=None, **kwargs):
 
 
 @multithreading_enabled
-def polygons(shells, holes=None, indices=None, **kwargs):
+def polygons(geometries, holes=None, indices=None, **kwargs):
     """Create an array of polygons.
 
     Parameters
     ----------
-    shell : array_like
-        An array of linearrings that constitute the out shell of the polygons.
-        Coordinates can also be passed, see linearrings.
+    geometries : array_like
+        An array of linearrings or coordinates (see linearrings).
+        If ``indices`` are given, the geometries include both the outer shell
+        and the holes. If not, the geometries only include the shells and the
```

Comment:
```suggestion
        and the holes; the first geometry for each index is the outer shell and all subsequent geometries in that index are the holes...
```

Reply:
Thanks ,I added your suggestion, at the `indices` description